src/devices/Fsr408/Fsr408.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+
+namespace Iot.Device.Fsr408
+{
+    public class Fsr408 : IDisposable
+    {
+        private int _pinNumber = 0;
+
+        private Mcp3008.Mcp3008 _adcConverter;
+        public int Resistance { get; set; } = 10000; // 10k ohm
+        public int PowerSupplied { get; set; } = 5000;  // 5 mV
+
+        public Fsr408(Mcp3008.Mcp3008 adcConverter)
+        {
+            _adcConverter = adcConverter;
+        }
+
+        public Fsr408(int pinNumber, Mcp3008.Mcp3008 adcConverter)
+        {
+            _adcConverter = adcConverter;
+            _pinNumber = pinNumber;
+        }
+
+        public int Read()
+        {
+            return _adcConverter.Read(_pinNumber);
+        }
+
+        public int ReadVoltage()
+        {
+            return CalculateVoltage(_adcConverter.Read(_pinNumber));
+        }
+
+        public int CalculateVoltage(int readValue)
```

Comment:
I'd make this private, and instead only expose the `ReadVoltage` method. Is there a scenario where you expect people to call `Read()` and then call `CalculateVoltage`? Why wouldn't they just call `ReadVoltage` instead? What is the advantage of exposing the raw Read value?

Reply:
Good question - @buyaa-n  ?